vllm/model_executor/layers/quantization/utils/fp8_utils.py

```diff
@@ -150,8 +146,7 @@ def input_to_float8(
     """This function quantizes input values to float8 values "
     "with tensor-wise quantization."""
     if dtype is None:
-        dtype = (torch.float8_e4m3fnuz
-                 if current_platform.is_rocm() else torch.float8_e4m3fn)
+        dtype = current_platform.fp8_dtype()
```

Comment:
Nit: could be `dtype = current_platform.fp8_dtype() if dtype is None else dtype` if it fits in a line